java/openjdk10/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+
+name             openjdk10
+version          10.0.1
+
+categories       java devel
+maintainers      {@breun breun.nl:nils} openmaintainer
+platforms        darwin
+license          GPL-2
+supported_archs  x86_64
+
+description      OpenJDK -- Open Java Development Kit
```

Comment:
No need to repeat the port name in the (short) description. So only keep what is after "--".

Reply:
Ok, updated.